src/main/scripts/org/reaktivity/specification/sse/challenge/respond.to.challenge/response.rpt

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright 2016-2019 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property serverTransport "nukleus://streams/target#0"
+
+accept ${serverTransport}
+       option nukleus:window 8192
+       option nukleus:transmission "half-duplex"
+accepted
+
+read nukleus:begin.ext ${http:beginEx()
+                             .typeId(nukleus:id("http"))
+                             .header(":scheme", "http")
+                             .header(":method", "GET")
+                             .header(":path", "/events/a8b7c6d5")
+                             .header(":authority", "localhost:8080")
+                             .header("accept", "text/event-stream")
+                             .build()}
+
+connected
+
+write nukleus:begin.ext ${http:beginEx()
+                              .typeId(nukleus:id("http"))
+                              .header(":status", "200")
+                              .header("content-type", "text/event-stream")
+                              .build()}
+
+write nukleus:challenge
+read "type:challenge\n\n"
+write flush
```

Comment:
Please add `${http:challengeEx().header(...).build()` to `write nukleus:challenge` and corresponding JSON to `read "type:challenge\n\n\data:JSON\n\n"`.
Also `write flush` may not be needed in this case - please remove if possible.